src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -64,47 +61,44 @@ public sealed class CodeAnalysisResultManager : IVsSolutionEvents, IVsUpdateSolu
             // Get temporary path for embedded files.
             TemporaryFilePath = Path.GetTempPath();
             TemporaryFilePath = Path.Combine(TemporaryFilePath, TemporaryFileDirectoryName);
+
+            IComponentModel componentModel = (IComponentModel)Package.GetGlobalService(typeof(SComponentModel));
+            if (componentModel != null)
+            {
+                this.sarifLocationTaggerService = componentModel.GetService<ISarifErrorListEventSelectionService>();
+            }
+
         }
 
         public static CodeAnalysisResultManager Instance = new CodeAnalysisResultManager(new FileSystem());
 
         public IDictionary<int, RunDataCache> RunIndexToRunDataCache { get; } = new Dictionary<int, RunDataCache>();
 
-        public int CurrentRunIndex { get; set; } = 0;
+        /// <summary>
+        /// Returns the last index given out by <see cref="GetNextRunIndex"/>.
+        /// </summary>
+        /// <remarks>
+        /// Currently on referenced for testing.
+        /// </remarks>
+        internal int CurrentRunIndex;
```

Comment:
>internal [](start = 8, length = 8)

this should be private.

Reply:
Still looks internal.

---
In reply to: [501895283](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r501895283) [](ancestors = 501895283)

Reply:
Yes, because the test references it.

---
In reply to: [501895283](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r501895283) [](ancestors = 501895283)